### PR TITLE
1657389 Add descriptions for SEO (series of related PRs)

### PR DESCRIPTION
**Pull request by TimShererWithAquent**
## Add descriptions for SEO

For anyone that GitHub notifies, this is for [1657389](https://dev.azure.com/mseng/TechnicalContent/_workitems/edit/1657389).

You're welcome but not required, according to KhairunJ's guidance, to review these proposed meta descriptions. I'll leave this PR open for 48 hours and address any feedback. After that, we'll sign off for merge. Thanks!

Please keep in mind our 100-160 char length for descriptions.

Fixes #1657389

### CODE REVIEW

**Comment by TimShererWithAquent** [comment]
There was a build failure. I'm going to close and reopen.

**Comment by TimShererWithAquent** [comment]
And reopen.

File: docs/csharp/programming-guide/concepts/linq/sample-xml-file-customers-and-orders-in-a-namespace.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 ---
 title: "Sample XML File: Customers and Orders in a Namespace2"
```
Comment (BillWagner) [inline docs/csharp/programming-guide/concepts/linq/sample-xml-file-customers-and-orders-in-a-namespace.md:2]: Not related to your PR. Let's delete the erroneous "2":

```suggestion
title: "Sample XML File: Customers and Orders in a Namespace"
```

**Review by BillWagner — CHANGES_REQUESTED** [review]
Thanks @TimShererWithAquent 

I have one requested change. I'll try and commit that and :shipit: it.

**Review by BillWagner — APPROVED** [review]

**Pull request merged**